code/unclassified/src/product_array.cpp

```diff
@@ -0,0 +1,70 @@
+/*Given an array arr[] of n integers, construct a Product Array prod[] (of same
+size) such that prod[i] is equal to the product of all the elements of arr[]
+except arr[i]. Solve it without division operator in O(n) time.*/
+
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+using namespace std;
```

Comment:
@HariniJeyaraman no need to use `using namespace std;` if you already used  `std::`. Also, change `cout` to `std::cout` if you are not using `using namespace std;`